pyproject.toml

```diff
@@ -7,7 +7,8 @@ requires = [
     "numpy==1.13.3; python_version=='3.6'",
     "numpy==1.14.5; python_version=='3.7'",
     "numpy==1.17.3; python_version=='3.8'",
+    "numpy==1.19.3; python_version=='3.9'",
```

Comment:
1.19.3 is also what is used at https://github.com/scipy/oldest-supported-numpy/commit/aa14bc2b75ed09865e20a91d5ee578a3493f7dd1